iglu-ruby-client.gemspec

```
@@ -33,8 +33,8 @@ Gem::Specification.new do |s|
     'wiki_uri'        => 'https://github.com/snowplow/iglu/wiki/Ruby-client'
   }
 
-  s.add_runtime_dependency "httparty", "<= 0.14.0"
+  s.add_runtime_dependency "httparty", "~> 0.15.0"
```

This locks down the version too tightly.

```suggestion
  s.add_runtime_dependency "httparty", "~> 0.15"
```